psql2mysql/__init__.py

```diff
@@ -194,16 +199,22 @@ class TargetDatabaseEmpty(Exception):
 
 
 class DbDataMigrator(object):
-    def __init__(self, config, source, target):
+    def __init__(self, config, source, target, chunk_size=None):
         self.cfg = config
         self.src_uri = source if source else cfg.CONF.source
         self.target_uri = target if target else cfg.CONF.target
+        try:
+            self.chunk_size = positive_int(chunk_size) \
+                if chunk_size is not None else cfg.CONF.chunk_size
+        except ValueError as e:
+            LOG.error(e.message)
```

Comment:
e.message is a Python 2ism, use LOG.error(str(e))

Reply:
This one is the reason for the travis 3.6 failures, good catch @s-t-e-v-e-n-k